backend/src/main/java/com/example/parkingspot/service/AuthService.java

```diff
@@ -0,0 +1,22 @@
+package com.example.parkingspot.service;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthService {
+
+  
+  private UserService userService;
+
+  public AuthService(UserService userService) {
+    this.userService = userService;
+  }
+
+  public String getAuth() {
+  Authentication auth = SecurityContextHolder.getContext().getAuthentication();
+  String username = auth.getName();
+  return userService.getUserId(username);
+  }
+}
```

Comment:
Nice reuseable service class. Method naming might become an issue down the road if we need something other then the userId